Documentation/Helm-Charts/ceph-cluster-chart.md.gotmpl

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
Could we name these template files instead so they still have the .md as the file extension? Otherwise, the editor doesn't have the highlighting when we're editing them during development. For example, ceph-cluster-chart-gotmpl.md. Hopefully the wildcards could still work with that. Also, I worry about someone editing the generated doc, then they lose their work when it is generated. Could we add a comment at the top of the doc that indicates it's generated (and should be hidden in the rendered docs)?

Reply:
I renamed the templates to `.gotmpl.md` and added a rule to exclude these too.